### ISSUE TITLE
Java Checker get's confused

### ISSUE THREAD

**Issue by eingorz** [MultiMC/MultiMC5#4125]
### Operating System

Bedrock Linux using artix linux openrc

### Description of bug

While launching an instance, instead of it launching, the console said that I need to download Java 16 which I already have but the Java Version Checker gets confused due to an unrelated error that slightly changes the format of what
`java -version` returns. 

The log says this:
Checking Java version...
Java checker returned some invalid data MultiMC doesn't understand:
[0.020s][warning][os,container] Duplicate cpuset controllers detected. Picking /sys/fs/cgroup/cpuset, skipping /bedrock/strata/bedrock/sys/fs/cgroup/cpuset.
os.arch=amd64
java.version=16.0.1
java.vendor=N/A

### Steps to reproduce

Not sure how someone would reproduce this.

### This issue is unique

- [X] I have searched the issue tracker and did not find an issue describing my bug.

**Comment by peterix** [comment]
You have something breaking potentially all applications and scripts on your system. Fix that. MultiMC is right in rejecting such broken Java.

**Comment by kumquat-ir** [comment]
This looks like java getting confused at some of the weird shit bedrock linux does to actually be able to function, not a broken system configuration

In this case, it seems to be the usual stratum mounting/linking/whatever shenanigans making java think there are multiple copies of the same file even though they're actually the same file

tl;dr bedrock linux is incredibly weird and causes strange errors sometimes, and this seems to be one of them, not sure if there's any good solution to this

**Comment by Gotchfutchian** [comment]
> This looks like java getting confused at some of the weird shit bedrock linux does to actually be able to function, not a broken system configuration
> 
> In this case, it seems to be the usual stratum mounting/linking/whatever shenanigans making java think there are multiple copies of the same file even though they're actually the same file
> 
> tl;dr bedrock linux is incredibly weird and causes strange errors sometimes, and this seems to be one of them, not sure if there's any good solution to this
> 
> edit: Might be able to be worked around on the user's end by some shenanigans with `strat`? I don't have a bedrock machine set up right now to test

Just tested with `strat`, java seems to ignore it and just bypasses it somehow. I think there should be a work around (question mark)?

**Comment by peterix** [comment]
Clearly something broken... It breaks stdout of Java processes.

**Comment by peterix** [comment]
Just imaging running a Java based CLI tool with this happening. Any such thing is broken on your system and you wouldn't be able to use such a tool in a shell properly.

**Comment by peterix** [comment]
This cannot be a 'feature'. It's more of a 'bug, elsewhere'.
